contrib/question-container-debugging.sh

```
@@ -76,8 +77,8 @@ END
 )
 
 echo_and_eval() {
-    echo "$@"
-    eval "$@"
+  echo "$@"
+  eval "$@"
```

Shellcheck (0.10.0)

[warning] eval negates the benefit of arrays. Drop eval to preserve whitespace/symbols (or eval as string).

(SC2294)